jsdoc/jsdoc
# Added in a man page for jsdoc

# Table
| Question                | Answer
| ---------------- | ---
| Bugfix?         | No
| New feature?     | Yes
| Breaking change? | No
| Deprecations?    | No
| Tests added?     | No
| Fixed issues     | N/A
| License          | Apache-2.0



# Overview
Added in a man page for `jsdoc`.
# List of Updates
Here is a list of updates
* Edited `package.json` to include an entry called `man` (`npm` automatically installs man pages inside the `man` entry)
* Created a file at `jsdoc.1`
* Added in a man page at `jsdoc.1` using `txt2man` on the `--help` output
# Testing
To test the layout, run:
```bash
git clone https://github.com/Yash-Singh1/jsdoc.git
cd jsdoc
man ./jsdoc.1
```
To test the changes, run:
```bash
git clone https://github.com/Yash-Singh1/jsdoc.git
cd jsdoc
sudo npm -g uninstall jsdoc ; npm link
```
Run `man jsdoc` and the manual page should appear.
To go back to the normal environment:
```bash
sudo npm -g uninstall jsdoc ; npm install -g jsdoc
```
***Note: The testing process will only work when the PR is open***
